ruby3.2-fluentd-1.16.yaml

```diff
@@ -64,6 +64,19 @@ pipeline:
              ${GEM_DIR}/*.md \
              ${GEM_DIR}/.github
 
+subpackages:
+  - name: ${{package.name}}-logging-operator-compat
+    description: Entrypoint used by the logging operator image
+    dependencies:
+      runtime:
+        - busybox
+    pipeline:
+      - runs: |
+          git clone https://github.com/kube-logging/fluentd-images.git
```

Comment:
Can you fetch or checkout a commit, just so this doesn't move forward unexpectedly?

Reply:
Good call, they don't have togs or releases so I just pinned to the current commit